README.md

```diff
@@ -36,7 +36,7 @@ Before you install DAHLIA, your system should have the following:
     - see [here](https://stackoverflow.com/a/19850273/260495) if you have issues installing `pg` gem with Postgres.app, you may need to use: `gem install pg -v 0.18.4 -- --with-pg-config=/Applications/Postgres.app/Contents/Versions/latest/bin/pg_config
 `
 1. `npm install` to install bower, grunt and other dependencies (which will also automatically `bower install` to load front-end JS libraries)
-1. `overcommit --install` to install git hooks into the repo
+1. Run `gem install overcommit` then `overcommit --install` to install git hooks into the repo
```

Comment:
looking at the gemfile, this should be there, but it's not getting installed in bundle install for me in this repo or in the LAP.

Reply:
I uninstalled all gems then re-installed bundle and re-ran bundle install and overcommit installed successfully this time. I went ahead and removed the extra text